src/opts.rs

```diff
@@ -266,23 +266,47 @@ pub enum Focus {
         String,
     ),
 
+    TestList(
+        // Show available tests (hidden: cargo shows the list as an error)
+        #[bpaf(long("test"), hide)]
+        (),
+    ),
+
```

Comment:
I think it should be possible to do something like this
```suggestion
    // Show available tests (hidden: cargo shows the list as an error)
    #[bpaf(long("test"), hide)]
    TestList
```

Reply:
Oh, that’s much nicer. Dunno why it didn’t occur to me.